my-first-react-app/src/index.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App.js';
+import './index.css';
+/*import Hello from './hello';*/
+
+
+/*class Hello extends React.Component {
+  render() {
+    return <h1>Hello {this.props.name}!</h1>;
+  }
+}
+
+class App extends React.Component {
+  render() {
+    const users = [
+      { _id: 1, name: 'Leia' },
+      { _id: 2, name: 'Luke' },
+      { _id: 3, name: 'Han' }
+    ];
+
+    return (
+      <div>
+        {
+          users.map(function(user) {
+            return <Hello key={user._id} name={user.name} />;
+          })
+        }
+      </div>
+    );
+  }
+}
+*/
```

Comment:
Remove obsolete code, if you need it, you can always get it back through commit history.